api/src/main/java/org/pmiops/workbench/api/UserAdminController.java

```diff
@@ -0,0 +1,132 @@
+  private final ActionAuditQueryService actionAuditQueryService;
+  private final UserDao userDao;
+  private final Provider<DbUser> userProvider;
+  private final Provider<WorkbenchConfig> workbenchConfigProvider;
+
+  public UserAdminController(
+      UserService userService,
+      ProfileService profileService,
+      ActionAuditQueryService actionAuditQueryService,
+      UserDao userDao,
+      Provider<DbUser> userProvider,
+      Provider<WorkbenchConfig> workbenchConfigProvider) {
+    this.userService = userService;
+    this.profileService = profileService;
+    this.actionAuditQueryService = actionAuditQueryService;
+    this.userDao = userDao;
+    this.userProvider = userProvider;
+    this.workbenchConfigProvider = workbenchConfigProvider;
+  }
+
+  @Override
+  @AuthorityRequired({Authority.ACCESS_CONTROL_ADMIN})
+  public ResponseEntity<EmptyResponse> bypassAccessRequirement(
+      Long userId, AccessBypassRequest bypassed) {
+    userService.updateBypassTime(userId, bypassed);
+    return ResponseEntity.ok(new EmptyResponse());
+  }
+
+  @Override
+  @AuthorityRequired({Authority.ACCESS_CONTROL_ADMIN})
+  public ResponseEntity<AdminUserListResponse> getAllUsers() {
+    return ResponseEntity.ok(
+        new AdminUserListResponse().users(profileService.getAdminTableUsers()));
+  }
+
+  @Override
+  @AuthorityRequired({Authority.RESEARCHER_DATA_VIEW})
+  public ResponseEntity<UserAuditLogQueryResponse> getAuditLogEntries(
+      String usernameWithoutGsuiteDomain,
+      Integer limit,
+      Long afterMillis,
+      @Nullable Long beforeMillis) {
+    final String username =
+        String.format(
+            "%s@%s",
+            usernameWithoutGsuiteDomain,
+            workbenchConfigProvider.get().googleDirectoryService.gSuiteDomain);
+    final long userDatabaseId = userService.getByUsernameOrThrow(username).getUserId();
+    final Instant after = Instant.ofEpochMilli(afterMillis);
+    final Instant before =
+        Optional.ofNullable(beforeMillis).map(Instant::ofEpochMilli).orElse(Instant.now());
+    return ResponseEntity.ok(
+        actionAuditQueryService.queryEventsForUser(userDatabaseId, limit, after, before));
+  }
+
+  /**
+   * Gets a JSON list of users and their registered tier access expiration dates.
+   *
+   * <p>This endpoint is intended as a temporary manual measure to assist with user communication
+   * during the rollout of Annual Access Renewal (AAR).
+   *
+   * <p>Once fully rolled out, we will have an automated expiration email process and this can
+   * likely be removed. See RW-6689 and RW-6703.
+   */
+  @Override
+  @AuthorityRequired({Authority.ACCESS_CONTROL_ADMIN})
+  public ResponseEntity<List<UserAccessExpiration>> getRegisteredTierAccessExpirations() {
+    return ResponseEntity.ok(userService.getRegisteredTierExpirations());
+  }
+
+  @Override
+  @AuthorityRequired({Authority.ACCESS_CONTROL_ADMIN})
+  public ResponseEntity<Profile> getUser(Long userId) {
```

Comment:
This also doesn't appear to be in use, and I don't see a good reason to support it. Going to remove this as well